src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package edu.cornell.kfs.rass.batch.service.impl;
+
+import edu.cornell.kfs.rass.batch.service.RassService;
+
+public class RassServiceImpl implements RassService {
+
+	@Override
+	public void readXML() {
+
+	}
+	
+	@Override
+	public boolean updateKFS() {
+		updateAgencies();
```

Comment:
Should we set a boolean like:
boolean successfullyProcessed = updateAgencies();
successfullyProcessed &= updateProposals();
successfullyProcessed &= updateAwards();
return successfullyProcessed;

Reply:
I will add the boolean.